File: android/app/src/main/java/com/campuswave/app/data/models/Banner.kt
package com.campuswave.app.data.models

data class Banner(
    val id: Int,
    val image_url: String,
    val created_at: String
)

data class BannerResponse(
    val banners: List<Banner>
)

data class AdminBannerUploadResponse(
    val message: String,
    val banner: Banner
)

File: backend/app/models/banner.py
from app.extensions import db
from datetime import datetime

class Banner(db.Model):
    __tablename__ = 'banners'
    
    id = db.Column(db.Integer, primary_key=True)
    image_url = db.Column(db.String(500), nullable=False)
    created_at = db.Column(db.DateTime, default=datetime.utcnow)
    
    def to_dict(self):
        return {
            'id': self.id,
            'image_url': self.image_url,
            'created_at': self.created_at.isoformat()
        }
